libclc/ptx-nvidiacl/libspirv/assert/__assert_fail.cl

```diff
@@ -1,14 +1,14 @@
 #include <spirv/spirv.h>
 
 void __assertfail(const char *__message, const char *__file, unsigned __line,
-                  const char *__function);
+                  const char *__function, size_t __charSize);
 
 _CLC_DECL void __assert_fail(const char *expr, const char *file,
                              unsigned int line, const char *func) {
-  __assertfail(expr, file, line, func);
+  __assertfail(expr, file, line, func, 1);
```

Comment:
Very minor and not that I would ever expect them to differ, but should this be

```suggestion
  __assertfail(expr, file, line, func, sizeof(char));
```

for good measure?

Reply:
Hi @steffenlarsen, sorry was off last couple of days, I would normally agree, but in the PTX interoperability doc, they make an explicit point:
> The only supported character size is 1